net/https-dns-proxy/Makefile

```diff
@@ -1,12 +1,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=https-dns-proxy
-PKG_VERSION:=2023-11-19
-PKG_RELEASE:=1
+PKG_VERSION:=2023.11.19
+PKG_RELEASE:=r1
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=https://github.com/aarond10/https_dns_proxy/
-PKG_SOURCE_DATE:=$(PKG_VERSION)
+PKG_SOURCE_DATE:=$(subst(.,-,$(PKG_VERSION)))
+PKG_SOURCE_RELEASE:=$(subst(r,,$(PKG_RELEASE)))
```

Comment:
@aparcar Is this wanted behavior? I think there will be several exceptions like this. 

While looking at your treewide commit in the main repo: https://github.com/openwrt/openwrt/commit/e8725a932e16eaf6ec51add8c084d959cbe32ff2

It sounds like we can drop those two lines as they are not required. Is that right?